src/main/java/io/codelex/polymorphism/practice/exercise6/AnimalFeedingMain.java

```diff
@@ -0,0 +1,69 @@
+package io.codelex.polymorphism.practice.exercise6;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class AnimalFeedingMain {
+
+    private static Food food = null;
+    private static Animal animal = null;
```

Comment:
this feels redundant, katru reizi kad tiek lasīts jauns  inout tad labāk veidot jaunu objektu
`setAnimal()` metodes vietā tad `createAnimal()` kurš tad viņu arī atgrieztu, savadāk tāds placeholder just looks a bit weird 😄

Reply:
Izlabots.